src/components/ChatLog.jsx

```diff
@@ -2,24 +2,25 @@ import ChatEntry from './ChatEntry.jsx';
 import PropTypes from 'prop-types';
 import './ChatLog.css';
 
-const ChatLog = ({ entries, onToggleLike }) => {
+const ChatLog = ({ entries, onToggleLike = () => {} }) => {
 	return (
 		<div className="chat-log">
 			{entries.map((entry) => (
 				<ChatEntry
           key={entry.id}
+          id={entry.id} // Pass the id directly
```

Comment:
👍 Passing the id all the way into the `ChatEntry` component lets it know which message it's working with.